app/controllers/posts_controller.rb

```diff
@@ -13,4 +13,27 @@ def show
       @post = nil
     end
   end
+
+  def new
+    @user = User.find(params[:user_id])
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(author_id: params[:user_id], **post_params)
+
```

Comment:
- [ ] @NicolasAndrehh great work on your new and create method, however, we are supposed to use the currently logged-in user (current_user) of the app using the `current_user`method we created

`hint`
```suggestion
  def new
    @post = Post.new
  end

  def create
  `@post = @current_user.posts.new(post_params)`

```

![image](https://github.com/NicolasAndrehh/Rails-Blog-App/assets/61408860/ae47ad77-7268-4c3f-8832-fcc720b613cf)add